gengokit/httptransport/templates/server.go

```diff
@@ -188,7 +197,7 @@ func errorEncoder(_ context.Context, err error, w http.ResponseWriter) {
 }
 
 type errorWrapper struct {
-	Error string ` + "`json:\"error\"`\n" + `
+	Error string ` + "`" + `json:"error"` + "`" + `
```

Comment:
What is this struct-tag-looking construct? This is a template file, but I still don't understand this, or why it could not just be
```
Error string `json:"error"`
```